lib/ts/recipe/emailverification/index.ts

```diff
@@ -38,22 +38,9 @@ export default class Wrapper {
     > {
         const recipeInstance = Recipe.getInstanceOrThrowError();
```

Comment:
- We probably don't need `getEmailForUserId` on a per recipe anymore cause we can directly use supertokens.getUser function if the config for this is not defined
- We should rename `getEmailForUserId` to `getEmailForRecipeUserId`